### DISCUSSION TITLE
REI Plugin Compatibilities' version management on Curse is a huge mess for 1.19

### DISCUSSION THREAD

**Discussion by cl0ck-byte** [shedaniel/RoughlyEnoughItems#1180] [Q & A] [answered]
![1.19](https://user-images.githubusercontent.com/35605833/198992270-af19f171-78e8-47de-a54e-615fbb75110e.png)

First we have v9.1.530 and now v9.0.23 has been released? I am confused on whichever version should I get for 1.19/1.19.2

**Comment by shedaniel** [comment, marked as answer]
Download the latest by date, sorry for this.

**Reply by cl0ck-byte** [reply]
No need to be sorry though, but thanks :-)
I hope to see more REI updates in the future
